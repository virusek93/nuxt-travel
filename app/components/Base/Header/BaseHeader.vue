<script setup lang="ts"></script>

<template>
  <nav class="flex min-h-20 items-center">
    <ul class="flex list-none gap-5 p-0">
      <li>
        <NuxtLink
          to="/"
          class="underline-offset-2 hover:underline"
          active-class="text-lime-600"
        >
          Home
        </NuxtLink>
      </li>
      <li>
        <NuxtLink
          to="/booking"
          class="underline-offset-2 hover:underline"
          active-class="text-lime-600"
        >
          Bookings
        </NuxtLink>
      </li>
      <li>
        <NuxtLink
          to="/travel"
          class="underline-offset-2 hover:underline"
          active-class="text-lime-600"
        >
          Travel
        </NuxtLink>
      </li>
    </ul>
  </nav>
</template>
