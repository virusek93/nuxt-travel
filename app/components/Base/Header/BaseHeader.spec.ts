import { describe, expect, it } from 'vitest'
import { mount } from '@vue/test-utils'
import BaseHeader from '~/components/Base/Header/BaseHeader.vue'

const global = {
  stubs: {
    RouterLink: true,
  },
}

describe('BaseHeader', () => {
  it('Should render and match the snapshot', () => {
    const wrapper = mount(BaseHeader, { global })
    expect(wrapper.element).toMatchSnapshot()
  })
})
