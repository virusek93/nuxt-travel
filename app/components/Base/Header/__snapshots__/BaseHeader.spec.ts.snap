// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BaseHeader > Should render and match the snapshot 1`] = `
<nav
  class="flex min-h-20 items-center"
>
  <ul
    class="flex list-none gap-5 p-0"
  >
    <li>
      <router-link-stub
        activeclass="text-lime-600"
        class="underline-offset-2 hover:underline"
        to="/"
      />
    </li>
    <li>
      <router-link-stub
        activeclass="text-lime-600"
        class="underline-offset-2 hover:underline"
        to="/booking"
      />
    </li>
    <li>
      <router-link-stub
        activeclass="text-lime-600"
        class="underline-offset-2 hover:underline"
        to="/travel"
      />
    </li>
  </ul>
</nav>
`;
